app/controllers/risks_controller.rb

```diff
@@ -80,21 +80,23 @@ def destroy
     end
   end
 
-
-private
+  private
 
   def permitted_params
-    params.require(:risk).permit(:title, :description, :probability, :impact, :risk_status_id)
+    params.require(:risk).permit(:title,
+                                 :description,
+                                 :controls,
+                                 :probability,
+                                 :impact,
+                                 :risk_status_id)
   end
 
   def find_risk
     @risk = Risk.includes(:project).find(params[:id])
   end
 
   def check_project_permission
-    if @risk && @risk.project != @project
-      render_403
-    end
+    return unless @risk && @risk.project != @project
```

Comment:
I think this method would read better like this:
`render_403 if @risk && @risk.project != @project`